src/Conferences/ConferencesHarvester.php

```diff
@@ -159,8 +163,8 @@ private function updateExistingConference(Conference $existingConference, Confer
     private function shouldBeIgnored(Conference $conference): bool
     {
         foreach ($this->getFilters() as $conferenceFilter) {
-            $filterName = $conferenceFilter->getName();
-            if (fnmatch($filterName, $conference->getName(), FNM_CASEFOLD) || \in_array($filterName, $conference->getTags())) {
+            $filterName = strtolower($conferenceFilter->getName());
+            if (str_contains(strtolower($conference->getName()), $filterName) || \in_array($filterName, array_map('strtolower', $conference->getTags()))) {
```

Comment:
`str_contains` has been introduced in PHP 8, we do not run on this version (yet ;) )

Reply:
@welcoMattic symfony ships polyfill, we can use it.